front/src/common/Map/utils.ts

```diff
@@ -26,10 +26,10 @@ export const onResultSearchClick = ({
   map,
   updateExtViewport,
   dispatch,
-  setSearch,
+  setSearchTerm,
   title,
 }: OnResultSearchClickType) => {
-  if (setSearch) setSearch(title);
+  if (setSearchTerm) setSearchTerm(title);
```

Comment:
Why do we need to update this state if we close the popup anyway ?

Reply:
Yes, thx!